<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevWidget</class>
 <widget class="QWidget" name="DevWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="vSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="hSplitterTop">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="BrowserWidget" name="lookBrowserWidget">
        <attribute name="title">
         <string>Looks</string>
        </attribute>
       </widget>
       <widget class="BrowserWidget" name="imageBrowserWidget">
        <attribute name="title">
         <string>Images</string>
        </attribute>
       </widget>
      </widget>
      <widget class="PipelineWidget" name="pipelineWidget" native="true"/>
      <widget class="QWidget" name="vLayout" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ImageWidget" name="imageWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="hSplitterBottom">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="OperatorListWidget" name="operatorListWidget" native="true"/>
      <widget class="QStackedWidget" name="operatorDetailWidget"/>
      <widget class="QWidget" name="scopeView" native="true">
       <layout class="QVBoxLayout" name="scopeVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="scopeHLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTabBar" name="scopeBar" native="true">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="scopeStack">
          <widget class="QWidget" name="scopeWidgetPage1"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowserWidget</class>
   <extends>QWidget</extends>
   <header>browserwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PipelineWidget</class>
   <extends>QWidget</extends>
   <header>pipelinewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header>qtabbar.h</header>
  </customwidget>
  <customwidget>
   <class>OperatorListWidget</class>
   <extends>QWidget</extends>
   <header>operatorlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
